gui/wxpython/location_wizard/wizard.py

```diff
@@ -559,7 +567,7 @@ def OnSearch(self, event):
         str = event.GetString()
         try:
             self.proj, self.projdesc = self.projlist.Search(
-                index=[0, 1], pattern=event.GetString())
+                index=[0, 1], pattern=str)
```

Comment:
Since you are editing this, it may be better to use different variable name instead of str, which is builtin function.

Reply:
Fixed here and another place with similar usage of `str`.